index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, minimum-scale=0.1">
+    <title>PORTFOLIO</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+        <img src="images/image_geometry_1.png" alt="">
+        <nav>
+            <div class="Button">≡</div>
+        </nav>
+    </header>
+    <main>
+        <h1 class="intro">
+            Javed <br> Shahbaz
+        </h1>
+        <section class="paragraph">
+            <p>
+            Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.
+            </p>
+         </section>    
+    </main>
```

Comment:
- [OPTIONAL] I would like to suggest that you should stick to the guiding template as close as possible. This involves font-size, font-weight, font-style, color, background-color, margin, and padding among others. Your design still need few adjustment for your project to look like the guiding template you chose. Kindly compare the two designs and work your design in line with the guiding template.

### Your Design

![your-design](https://user-images.githubusercontent.com/26597432/200136815-8bbb7b7f-ca4d-4670-bdf2-e60a2e8ed8bc.png)


### Figma Design

![temp](https://user-images.githubusercontent.com/26597432/200136827-30aebbcc-1ba0-4355-8672-62c4c09541cb.png)